### ISSUE TITLE
Permission 출력 관련 질문드립니다.

### ISSUE THREAD

**Issue by YueunLee** [SNU-ARC/2024_spring_sysprog_Lab2#10]
ls 실행 시와 다르게, setuid 비트 등 특수 권한의 경우 dirtree 구현 시에는 무시하고 출력하면 될까요? 실제로는 rwsrwsrwt 이더라도 rwxrwxrwx 이렇게만 출력하면 되는지 궁금합니다.

**Comment by yunjayh** [comment]
네. User owner, Group owner, Other 의 permission만 말씀해주신대로 출력해주시면 됩니다.
